src/main/java/authentication/SymExtensionAppRSAAuth.java

```diff
@@ -122,15 +142,17 @@ public AppAuthResponse appAuthenticate() {
         return null;
     }
 
-    private PrivateKey getPrivateKey() {
-        PrivateKey privateKey = null;
-        try {
-            privateKey = JwtHelper.parseRSAPrivateKey(
-                new File(config.getAppPrivateKeyPath() + config.getAppPrivateKeyName()));
-        } catch (IOException | GeneralSecurityException e) {
-            logger.error(e.getMessage());
+    private PrivateKey getAppPrivateKey() {
```

Comment:
Even though this is not  in scope of the PR, this is a private method so it should not have any impact.
Moreover, it gives actually context to the private key.